help.go

```diff
@@ -28,7 +28,9 @@ COMMANDS:
    {{end}}{{if .Flags}}
 GLOBAL OPTIONS:
    {{range .Flags}}{{.}}
-   {{end}}{{end}}
+   {{end}}{{end}}{{if .Copyright }}
+COPYRIGHT:
+   {{.Copyright}}{{end}}
 `
```

Comment:
Interesting, I didn't realize that `with` did that (or that it had an `else` clause). That is nicer.